src/components/pool-display.tsx

```
@@ -84,6 +90,135 @@ export function PoolDisplay({ poolId = 0n }: PoolDisplayProps) {
+    userInfo.refetch();
+    pendingRewards.refetch();
+    allowance.refetch();
+    balance.refetch();
+  };
+
+  if (
+    stakingSettings.error ||
+    lpTokenInfo.error ||
+    rewardTokenInfo.error ||
+    poolInfo.error ||
+    (account.isConnected && (pendingRewards.error || userInfo.error || allowance.error || balance.error))
```

Biome (2.3.13)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)